homework-classes/Contributor.js

```diff
@@ -14,6 +13,21 @@ class Contributor {
    */
   render(container) {
     // TODO: replace the next line with your code.
-    Util.createAndAppend('pre', container, { text: JSON.stringify(this.contributor, null, 2) });
+    const contributorsDiv = Util.createAndAppend('div', root, { id: 'contributors' });
+    for (let contributor of listOfContributors) {
```

Comment:
No need for loop here. One class, will be for one contributor. Can delete this line.